src/kinova_gen3_mujoco_config/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.22)
+project(kinova_gen3_mujoco_config)
+
+find_package(ament_cmake REQUIRED)
```

Comment:
Paul wrote some CMake code to ensure that MuJoCo is installed in his Panda config. We probably want to do something similar here. If this CMake code is something that is going to be used across multiple configs (i.e., any MuJoCo config), we probably want to put it into a CMake module so that it can be re-used easily elsewhere

Reply:
I think those details are actually in the picknik_mujoco_ros CMakeLists file. But I added an exec_depend on that project in the package.xml of this config package